homeassistant/components/home_connect/api.py

```diff
@@ -184,6 +188,7 @@ def get_program_sensors(self):
                 ATTR_ICON: icon,
                 ATTR_DEVICE_CLASS: device_class,
                 ATTR_SIGN: sign,
+                ATTR_TRANSLATION_KEY: None,
```

Comment:
This should be migrated to use `EntityDescription`s instead.

Reply:
Hi Frenck,
Help me out here.. On purpose I didn't do any refactoring in this PR and only extended functionality. This conform as you requested on another PR to not mix changes in Feature and refactoring in a PR.
It's one or the other...

Reply:
I think we should consider refactoring first, before adding new things.